odl/trafos/backends/pywt_bindings.py

```diff
@@ -24,21 +25,20 @@
 
 
 __all__ = ('PAD_MODES_ODL2PYWT', 'PYWT_SUPPORTED_MODES', 'PYWT_AVAILABLE',
-           'pywt_wavelet', 'pywt_pad_mode', 'pywt_coeff_shapes',
-           'pywt_flat_coeff_size', 'pywt_max_nlevels',
-           'pywt_flat_array_from_coeffs', 'pywt_coeffs_from_flat_array',
-           'pywt_single_level_decomp', 'pywt_single_level_recon',
-           'pywt_multi_level_decomp', 'pywt_multi_level_recon')
+           'pywt_wavelet', 'pywt_pad_mode', 'precompute_raveled_slices')
 
 
+# A clear illustration of all of these padding modes is available at:
+# https://pywavelets.readthedocs.io/en/latest/ref/signal-extension-modes.html
```

Comment:
Okay. There is also some documentation in the init of `WaveletTransformBase` that I neglected to add the new edge modes to. 

I also just noticed that there is an error in the description of the mode `symmetric`.  In the text [here](https://github.com/odlgroup/odl/blob/553f9fd8eb619a248baefb995550b6e399a7b845/odl/trafos/wavelet.py#L81) it says that symmetric does not duplicate the outermost values, however in `PyWavelets` it does. The mode `reflect` in PyWavelets is the one that has the described behaviour. Should I change the dictionary so that the ODL name `symmetric` points to PyWavelets `reflect` for consistency with ODL convention or should I just fix the description in the docstring? If the former, what would we name the PyWavelets `symmetric` mode in that case? Matlab refers to these two cases as whole-sample symmetric (`symw`) and half-sample symmetric (`symh`).

Reply:
That was error on our side, see also #645. If you could fix that, it'd be great.